higher_health/views.py

```diff
@@ -2,16 +2,17 @@
 from django.shortcuts import render
 
 from .forms import HealthCheckLogin, HealthCheckQuestionnaire
+from .utils import get_risk_level
 
 
 def healthcheck_questionnaire(request):
     submitted = False
     if request.method == "POST":
         form = HealthCheckQuestionnaire(request.POST)
         if form.is_valid():
-            # cd = form.cleaned_data
-            # assert False
-            return HttpResponseRedirect("/?submitted=True")
+            cd = form.cleaned_data
+            risk_level = get_risk_level(cd)
+            return HttpResponseRedirect(f"/?submitted=True&risk_level={risk_level}")
```

Comment:
Eventually we'll need to add the date to this results screen, so we might need to rather store this in the context. But happy to do this for now.

Reply:
@erikh360 No, this won't work since just refreshing the page results in multiple submission. We'll need to ensure that when you refresh the page, it brings that popup that says "the form was submitted, are you sure you want to submit again"